tomcat/src/main/java/nextstep/jwp/controller/LoginController.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.jwp.controller;
+
+import static org.apache.coyote.http11.handler.StaticResourceHandler.readFile;
+
+import java.io.IOException;
+import nextstep.jwp.db.InMemoryUserRepository;
+import nextstep.jwp.exception.InvalidPasswordException;
+import nextstep.jwp.exception.UserNotFoundException;
+import nextstep.jwp.model.User;
+import org.apache.coyote.http11.Http11Processor;
+import org.apache.coyote.http11.handler.AbstractController;
+import org.apache.coyote.http11.request.HttpRequest;
+import org.apache.coyote.http11.response.HttpResponse;
+import org.apache.coyote.http11.session.Session;
+import org.apache.coyote.http11.session.SessionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoginController extends AbstractController {
+
+    private static final Logger log = LoggerFactory.getLogger(Http11Processor.class);
+
+    @Override
+    protected HttpResponse doPost(final HttpRequest request) {
+        try {
+            final HttpResponse response = HttpResponse.found(request.getHttpVersion(), request.getContentType(),
+                    "/index.html");
+            login(request, response);
+            return response;
+        } catch (RuntimeException e) {
+            return HttpResponse.found(request.getHttpVersion(), request.getContentType(), "/401.html");
+        }
+    }
+
+    private void login(final HttpRequest request, final HttpResponse response) {
+        final User user = InMemoryUserRepository.findByAccount(request.getRequestBodyValue("account"))
+                .orElseThrow(UserNotFoundException::new);
+        final boolean isCorrectPassword = user.checkPassword(request.getRequestBodyValue("password"));
+        if (!isCorrectPassword) {
+            throw new InvalidPasswordException();
+        }
```

Comment:
a : password를 변수로 빼는편이 더 가독성이 좋아보여요 ㅎㅎ

```suggestion
        final String password = request.getRequestBodyValue("password");
        if (user.checkPassword(password)) {
            throw new InvalidPasswordException();
        }
```

Reply:
변수로 빼는게 더 가독성 좋아보이네요! 변경했습니다👍